docs/errors.md

```diff
@@ -35,35 +38,56 @@ type Error interface {
-InvalidArgument    | "invalid_argument"    | 400 BadRequest
-DeadlineExceeded   | "deadline_exceeded"   | 408 RequestTimeout
-NotFound           | "not_found"           | 404 Not Found
-BadRoute           | "bad_route"           | 404 Not Found
-AlreadyExists      | "already_exists"      | 409 Conflict
-PermissionDenied   | "permission_denied"   | 403 Forbidden
-Unauthenticated    | "unauthenticated"     | 401 Unauthorized
-ResourceExhausted  | "resource_exhausted"  | 403 Forbidden
-FailedPrecondition | "failed_precondition" | 412 Precondition Failed
-Aborted            | "aborted"             | 409 Conflict
-OutOfRange         | "out_of_range"        | 400 Bad Request
-Unimplemented      | "unimplemented"       | 501 Not Implemented
-Internal           | "internal"            | 500 Internal Server Error
-Unavailable        | "unavailable"         | 503 Service Unavailable
-DataLoss           | "dataloss"            | 500 Internal Server Error
-NoError            | ""                    | 200 OK
-```
-
-The most common ErrorCodes are probably `InvalidArgument`, `NotFound` and `Internal`.
-
-Documentation for each ErrorCode is in [the godoc page](https://godoc.org/github.com/twitchtv/twirp#ErrorCode).
+Each error code is defined by a constant in the `twirp` package:
+
+| twirp.ErrorCode    | JSON/String         |  HTTP status code
+| ------------------ | ------------------- | ------------------
+| Canceled           | canceled            | 408 RequestTimeout
+| Unknown            | unknown             | 500 Internal Server Error
+| InvalidArgument    | invalid_argument    | 400 BadRequest
+| DeadlineExceeded   | deadline_exceeded   | 408 RequestTimeout
+| NotFound           | not_found           | 404 Not Found
+| BadRoute           | bad_route           | 404 Not Found
+| AlreadyExists      | already_exists      | 409 Conflict
+| PermissionDenied   | permission_denied   | 403 Forbidden
+| Unauthenticated    | unauthenticated     | 401 Unauthorized
+| ResourceExhausted  | resource_exhausted  | 403 Forbidden
+| FailedPrecondition | failed_precondition | 412 Precondition Failed
+| Aborted            | aborted             | 409 Conflict
+| OutOfRange         | out_of_range        | 400 Bad Request
+| Unimplemented      | unimplemented       | 501 Not Implemented
+| Internal           | internal            | 500 Internal Server Error
+| Unavailable        | unavailable         | 503 Service Unavailable
+| DataLoss           | dataloss            | 500 Internal Server Error
+
+For more information on each code, see the [Errors Spec](spec_v5.md).
+
+### HTTP Errors from Intermediary Proxies
+
+It is also possible for Twirp Clients to receive non-200 responses with invalid 
+Twirp Errors (non JSON, or with invalid keys) that may be generated by 
+intermediary proxies like a load balancer. Twirp clients return valid Twirp 
+errors depending on the HTTP status of the invalid responses:
```

Comment:
Rephrase to 

"It is also possible for Twirp Clients to receive HTTP responses with non-200 status codes but without an expected error message. For example, proxies or load balancers might return a "503 Service Temporarily Unavailable" body, which cannot be deserialized into a Twirp error.

In these cases, generated Go clients will return `twirp.Error`s with a code which depends upon the HTTP status of the invalid response:"